src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -190,6 +193,10 @@ public void loadGeppettoProject(String requestID, IGeppettoProject geppettoProje
 			setConnectionProject(geppettoProject);
 			websocketConnection.sendMessage(requestID, OutboundMessages.PROJECT_LOADED, projectJSON);
 
+			boolean persisted = !geppettoProject.isVolatile();
+			String update = "{\"persisted\":" + persisted + ",\"projectId\":" + geppettoProject.getId()  +"}";
```

Comment:
@slarson Those messages are JSON types, which could be made other ways, GSON api might be one of them. We have this JSON messages everywhere on this class, would be a few hours refactor it we got rid of them.